functions.php

```diff
@@ -522,43 +522,43 @@ function get_top_story_details() {
  * @return array
  * @author Chris Conover
  **/
-function get_featured_stories_details() {
+function get_featured_stories_details( $limit = 3 ) {
```

Comment:
Should we change this default limit value to 4 since that would be an even placement with having 2 stories in each row?

Reply:
I set the default to three to match the number of items in the old email. I am changing it to two since only two are available in the old email. :)